src/types.ts

```diff
@@ -1956,7 +1962,7 @@ export type Policy = {
   owner?: boolean;
   priority?: number;
   resources?: string[];
-  roles?: string[];
+  roles?: string[] | null;
```

Comment:
isn't `?` means it's nullable already?

Reply:
`?` means `string[] | undefined` which is a bit different from `null`. So the result type here is `string[] undefined | null`

Reply:
> yeah but does it make sense here?

That's a good question :) I don't think this should be necessary

Reply:
not to me ... but backend sent `null` in one of our test case

Reply:
I can remove it if we can ensure it never null from backend response